src/components/about/partners/partners-styles.module.scss

```diff
@@ -18,25 +19,23 @@
 }
 
 .title {
+  @extend %headline;
   color: $white;
-  font-family: $font-family-1;
-  font-size: $font-size-xl;
-  font-weight: unset;
-  letter-spacing: 1px;
-  line-height: 30px;
   margin: 0;
   margin-bottom: 50px;
   text-shadow: 0 0 10px 0 rgba(255,255,255,0.5);
 }
 
 .logosWrapper {
   display: flex;
+  justify-content: center;
```

Comment:
After this change, the logos are justified to the center, but according to the design they should be justified to the left:
![Screenshot from 2019-06-21 09 55 48](https://user-images.githubusercontent.com/15097138/59910917-c78cfb80-940a-11e9-9881-b53da9c1621b.png)

I'll suggest to remove it and use a theme class in `partners.js`:
![image](https://user-images.githubusercontent.com/15097138/59910997-f0ad8c00-940a-11e9-9b96-3403adc47c04.png)
You can pass this way a custom class and e.g. justify content to the center:
![Screenshot from 2019-06-21 09 58 39](https://user-images.githubusercontent.com/15097138/59912415-e214a400-940d-11e9-865c-fe61fb5338dc.png)